cdm/core/src/main/java/ucar/nc2/ft2/coverage/HorizCoordSys.java

```diff
@@ -337,7 +337,7 @@ private Optional<CoverageCoordAxis> subsetLon(LatLonRect llbb, int stride) {
     double end = lonAxis.getEndValue();
 
     // use MAMath.MinMax as a container for two values, min and max
-    List<MAMath.MinMax> lonIntvs = subsetLonIntervals(wantMin, wantMax, start, end);
+    List<MAMath.MinMax> lonIntvs = subsetLonIntervals(wantMin, wantMax, start, end, llbb.containsAllLongitude());
```

Comment:
Can you add a comment somewhere that llbb = latlonboundingbox? I'm trying to get in the habit of labeling these things since we keep ending up going ??? about them haha